ServerlessApplication/README.md

```diff
@@ -0,0 +1,22 @@
+# Overview
+This repo contains the AWS artifacts required for integrating with the Salesforce Service Cloud Voice (SCV) product.
+
+## Build, Deploy, and Publish AWS Lambda Functions
+**Prerequisite** - Install gradle:
+```
+brew install gradle
+```
+**Step 1** - Build the serverless application:
+```
+cd ..
```

Comment:
why? cd ..

Reply:
Because this is in the one folder inside.